lib/src/main/java/scal/io/liger/view/Util.java

```diff
@@ -64,10 +64,14 @@ public static void startPublishActivity(Activity host, StoryPath storyPath) {
                 }
                 num++;
             }
-            if (message.length() > 0) {
+            if (message.length() > 3) {
                 Toast.makeText(host, "Clips " + message + "have a duration of 0", Toast.LENGTH_LONG).show();
                 Log.e("Util", "Clips " + message + "have a duration of 0");
                 return;
+            } else if (message.length() > 0) {
+                Toast.makeText(host, "Clip " + message + "has a duration of 0", Toast.LENGTH_LONG).show();
```

Comment:
all human visible messages like these must be in strings.xml so we can translate them